src/tokens/TokenDetails.tsx

```diff
@@ -145,24 +159,11 @@ function Actions({ token }: { token: TokenBalance }) {
   const isSwapEnabled = useSelector(isAppSwapsEnabledSelector)
```

Comment:
We might need a new feature gate for this as currently the only time swaps would display in more actions for a supported token is when swaps are also displayed in the hamburger drawer.

Reply:
Why do we need a new feature gate? I believe the remote config, even though is called `showSwapMenuInDrawerMenu`, is actually just to used to turn on / off swap action in the home screen actions. We no longer show swap in the drawer menu.

Reply:
`showSwapMenuInDrawerMenu` if set to false shows swap actions in the home screen action carousel. Then in `src/tokens/TokenDetails.tsx` we only display swaps as an action if `showSwapMenuInDrawerMenu` is true. If we ever want to display swaps in the home actions and in the token actions. Aside from that I'd argue that setting `showSwapMenuInDrawerMenu` to false to make it display in the home carousel is not very clear based on the remote config name. This could be addressed in a follow up PR.

Reply:
Got the double negatives mixed up 🤦. You're right this is essentially just a swaps enabled feature gate.